### PR TITLE
Fix unicode in requests

### PR DESCRIPTION
**Pull request by nowakowski**

### CODE REVIEW

**Comment by mkasztelnik** [comment]
@nowakowski  what about hound errors?

File: lib/handle_invalid_percent_encoding.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    begin
+      # calling env.dup here prevents bad things from happening
+      request = Rack::Request.new(env.dup)
+      # calling request.params is sufficient to trigger the error
+      # see https://github.com/rack/rack/issues/337#issuecomment-46453404
+      request.params
+      @app.call(env)
+    rescue ArgumentError => e
+      raise unless e.message =~ /invalid %-encoding/
+      @logger.debug("Wywoływane!")
```
Comment (mkasztelnik) [inline lib/handle_invalid_percent_encoding.rb]: Please remove this line

File: lib/handle_invalid_percent_encoding.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      render_json_error('Wrong encoding', status: :forbidden, type: :forbidden)
```
Comment (mkasztelnik) [inline lib/handle_invalid_percent_encoding.rb]: Why `forbidden` not `bad_request`?

Comment (nowakowski) [inline lib/handle_invalid_percent_encoding.rb]: Good catch.

File: lib/handle_invalid_percent_encoding.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+require 'atmosphere/json_error_handler'
+require 'rack'
+require 'logger'
+class HandleInvalidPercentEncoding
+  include Atmosphere::JsonErrorHandler
+
+  DEFAULT_CONTENT_TYPE = 'text/html'
+  DEFAULT_CHARSET = 'utf-8'
```
Comment (mkasztelnik) [inline lib/handle_invalid_percent_encoding.rb]: This constants are not used anymore - can be removed.

**Comment by mkasztelnik** [comment]
:+1:

**Pull request merged**